### ISSUE TITLE
It seems that lnmessage v0.2.4 breaks `LnMessage.disconnect()` method

### ISSUE THREAD

**Issue by tonyaldon** [aaronbarnardsound/lnmessage#42]
I think that `LnMessage.disconnect()` produces the error (that I reproduce below) because when I removed the line `ln.disconnect()` in the following script I no longer get the error.

Note that with v0.2.2 and v0.2.3 there's no error.

`getinfo.js`:

```js
#!/usr/bin/env node

import LnMessage from 'lnmessage'
import net from 'net'

// node l1
const NODE_ID = '02364c1ae32588bf4b94d17770b60a7cfcd59042bf6eeec7fea867b5c914c61b73'
const NODE_IP = '127.0.0.1'
const NODE_PORT = '7171'
// rune
const RUNE = 'Z99SUjfyevTx3DDNMCxVlfyyiQrzIdorBjPwMqXNQ-c9MA=='

const ln = new LnMessage({
  remoteNodePublicKey: NODE_ID,
  tcpSocket: new net.Socket(),
  ip: NODE_IP,
  port: NODE_PORT
})

await ln.connect()

const getinfo = await ln.commando({
  method: 'getinfo',
  params: [],
  rune: RUNE
})

console.log(getinfo)

ln.disconnect()
```

I run this script locally on a regtest node using `startup_regtest.sh` script from `lightning` repo to setup the regtest network:

    $ lightningd --version
    v23.05.2
    $ source ./lightning/contrib/startup_regtest.sh
    $ start_ln
    $ l1-cli getinfo
    {
       "id": "02364c1ae32588bf4b94d17770b60a7cfcd59042bf6eeec7fea867b5c914c61b73",
       "alias": "BIZARRESET",
       "color": "02364c",
       "num_peers": 0,
       "num_pending_channels": 0,
       "num_active_channels": 0,
       "num_inactive_channels": 0,
       "address": [],
       "binding": [
          {
             "type": "ipv4",
             "address": "127.0.0.1",
             "port": 7171
          }
       ],
       "version": "v23.05.2",
       "blockheight": 1,
       "network": "regtest",
       "fees_collected_msat": 0,
       "lightning-dir": "/tmp/l1-regtest/regtest",
       "our_features": {
          "init": "08a0000a0269a2",
          "node": "88a0000a0269a2",
          "channel": "",
          "invoice": "02000002024100"
       }
    }
    $ l1-cli commando-rune
    {
       "rune": "Z99SUjfyevTx3DDNMCxVlfyyiQrzIdorBjPwMqXNQ-c9MA==",
       "unique_id": "0",
       "warning_unrestricted_rune": "WARNING: This rune has no restrictions! Anyone who has access to this rune could drain funds from your node. Be careful when giving this to apps that you don't trust. Consider using the restrictions parameter to only allow access to specific rpc methods."
    }
    $ ./getinfo.js
    {
      id: '02364c1ae32588bf4b94d17770b60a7cfcd59042bf6eeec7fea867b5c914c61b73',
      alias: 'BIZARRESET',
      color: '02364c',
      num_peers: 1,
      num_pending_channels: 0,
      num_active_channels: 0,
      num_inactive_channels: 0,
      address: [],
      binding: [ { type: 'ipv4', address: '127.0.0.1', port: 7171 } ],
      version: 'v23.05.2',
      blockheight: 1,
      network: 'regtest',
      fees_collected_msat: 0,
      'lightning-dir': '/tmp/l1-regtest/regtest',
      our_features: {
        init: '08a0000a0269a2',
        node: '88a0000a0269a2',
        channel: '',
        invoice: '02000002024100'
      }
    }
    file:///home/tony/clnlive/node_modules/lnmessage/dist/chacha/chacha20.js:18
            this.input[4] = key.readUInt32LE(0);
                                ^
    
    TypeError: Cannot read properties of undefined (reading 'readUInt32LE')
        at new Chacha20 (file:///home/tony/clnlive/node_modules/lnmessage/dist/chacha/chacha20.js:18:29)
        at new Cipher (file:///home/tony/clnlive/node_modules/lnmessage/dist/chacha/index.js:15:23)
        at createCipher (file:///home/tony/clnlive/node_modules/lnmessage/dist/chacha/index.js:125:12)
        at ccpEncrypt (file:///home/tony/clnlive/node_modules/lnmessage/dist/crypto.js:46:20)
        at NoiseState.encryptMessage (file:///home/tony/clnlive/node_modules/lnmessage/dist/noise-state.js:301:20)
        at LnMessage._sendPingMessage (file:///home/tony/clnlive/node_modules/lnmessage/dist/index.js:165:33)
        at listOnTimeout (node:internal/timers:569:17)
        at process.processTimers (node:internal/timers:512:7)
    
    Node.js v18.17.0

Tell me if I'm doing something wrong or if you need more details.

**Comment by lnbc1QWFyb24** [comment]
Ah I just realised that I do not clear the ping timeout on disconnect which I think might be causing this issue. I'll get a fix happening today!

**Comment by lnbc1QWFyb24** [comment]
Hey @tonyaldon I released a fix in [v0.2.5](https://github.com/aaronbarnardsound/lnmessage/releases/tag/0.2.5)
Let me know if that fixes it on your end and then we can close this issue. Thanks for reporting!

**Comment by tonyaldon** [comment]
Hey @aaronbarnardsound, I've just tested the same script with v0.2.5 and I got the same error.

**Comment by lnbc1QWFyb24** [comment]
Ah there was another bug specifically for TCP socket connections. I should have directly replicated your issue! Should be fixed in [v0.2.6](https://github.com/aaronbarnardsound/lnmessage/releases/tag/0.2.6). Let me know if that looks good to you.

**Comment by tonyaldon** [comment]
v0.2.6 fixes the bug!  Thanks
